android/src/main/java/com/getcapacitor/community/admob/rewarded/AdRewardExecutor.java

```diff
@@ -39,6 +40,16 @@ public void prepareRewardVideoAd(final PluginCall call, BiConsumer<String, JSObj
                     try {
                         final AdRequest adRequest = RequestHelper.createRequest(adOptions);
                         final String id = AdViewIdHelper.getFinalAdId(adOptions, adRequest, logTag, contextSupplier.get());
+
+                        if (call.getString("userId", "") != "" || call.getString("customData", "") != "") {
```

Comment:
You do not need to use `call.getXYZ` Here since it is already part of your `adOptions` object. You can just do 
```
if (adOptions.userId != "" || adOptions.customData != "")
```